README.md

```diff
@@ -649,7 +648,7 @@ Array
 Woohoo Labs. Yin was designed to make error handling as easy and customizable as possible. That's why
 all the default exceptions extend the `JsonApiException` class and contain
 an [error document](#documents-for-error-responses) with the appropriate error object(s). However, if you
-only want to respond with an error document in case of an exception, you only need to do this:
+only want to respond with an error document in case of an exception, you need only do this:
```

Comment:
This change is stylistic. As a general rule, using infinitives muddies up the sentences, as it adds an additional word ('to').

Again, both versions are correct, its just a matter of preference! I think mine clears it up a bit, but by all means, go with the phrasing you find familiar. :)